extension/src/popup/Router.tsx

```diff
@@ -243,7 +243,7 @@ export const Router = () => {
         <PublicKeyRoute path={ROUTES.unlockBackupPhrase}>
           <UnlockBackupPhrase />
         </PublicKeyRoute>
-        <PublicKeyRoute path={ROUTES.settings}>
+        <PublicKeyRoute path={ROUTES.settings} exact>
```

Comment:
@acharb I found that this is needed if you want to nav to the parent of a nested route (in this case, nav'ing to `/settings/` rather than `/settings/preferences`)